static/js/portico/signup.js

```diff
@@ -230,4 +233,36 @@ $(() => {
     $("body").on("click", "#choose_email .choose-email-box", function () {
         this.parentNode.submit();
     });
+
+    $("#new-user-email-address-visibility .change_email_address_visibility").on("click", () => {
+        Micromodal.show("change-email-address-visibility-modal", {
+            disableFocus: true,
+            openClass: "modal--opening",
+        });
+    });
+
+    $("#change-email-address-visibility-modal .dialog_submit_button").on("click", () => {
+        const selected_val = Number.parseInt($("#new_user_email_address_visibility").val(), 10);
+        $("#email_address_visibility").val(selected_val);
+        Micromodal.close("change-email-address-visibility-modal");
+
+        let selected_option_text;
```

Comment:
Let's add a comment here noting these strings are duplicated in the template, and should be kept in sync.

Reply:
Or actually, could we make this text into a small Handlebars template taking an argument `option_code` that can be rendered here and included in the other place? Not sure if it makes things any simpler, since we'd need to pass the `settings_config` data in as a parameter. So up to you how to handle this.

Reply:
I am not sure whether we can or how to include a handlebars template inside an html file. So, added a comment for now.

Reply:
Yeah that seems fine, I forgot this was still a Django template. I don't think we've really done much with handlebars outside the app.